From: Ira Weiny <ira.weiny@intel.com>
Date: Thu, 28 Jul 2016 12:27:29 -0400
Subject: [PATCH 153/296] IB/hfi1: Add parameter names to callback declarations
Patch-mainline: v4.8-rc1
Git-commit: 862548dace34690b6a477f32e8ce68b50f7cbdf6
References: FATE#321231 FATE#321473

This makes it more clear what these functions are
operating on.

Reviewed-by: Dean Luick <dean.luick@intel.com>
Signed-off-by: Ira Weiny <ira.weiny@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/hfi1/mmu_rb.h | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/drivers/infiniband/hw/hfi1/mmu_rb.h b/drivers/infiniband/hw/hfi1/mmu_rb.h
index 215c728..45e7245 100644
--- a/drivers/infiniband/hw/hfi1/mmu_rb.h
+++ b/drivers/infiniband/hw/hfi1/mmu_rb.h
@@ -57,11 +57,12 @@ struct mmu_rb_node {
 };
 
 struct mmu_rb_ops {
-	bool (*filter)(struct mmu_rb_node *, unsigned long, unsigned long);
-	int (*insert)(struct rb_root *, struct mmu_rb_node *);
-	void (*remove)(struct rb_root *, struct mmu_rb_node *,
-		       struct mm_struct *);
-	int (*invalidate)(struct rb_root *, struct mmu_rb_node *);
+	bool (*filter)(struct mmu_rb_node *node, unsigned long addr,
+		       unsigned long len);
+	int (*insert)(struct rb_root *root, struct mmu_rb_node *mnode);
+	void (*remove)(struct rb_root *root, struct mmu_rb_node *mnode,
+		       struct mm_struct *mm);
+	int (*invalidate)(struct rb_root *root, struct mmu_rb_node *node);
 };
 
 int hfi1_mmu_rb_register(struct rb_root *root, struct mmu_rb_ops *ops);
-- 
1.8.5.6

